libs/community/langchain_community/vectorstores/milvus.py

```diff
@@ -374,6 +378,7 @@ def _create_collection(
             fields,
             description=self.collection_description,
             partition_key_field=self._partition_key_field,
+            enable_dynamic_field=self.enable_dynamic_field
```

Comment:
If `enable_dynamic_field` is not passed in, `CollectionSchema` assigns `self._enable_dynamic_field = None`. Here we are changing it to False in the general case. Will any behavior change?